wiki/Split_large_file.md

```diff
@@ -40,84 +40,83 @@ def batch_iterator(iterator, batch_size) :
     entries from the supplied iterator.  Each list will have
     batch_size entries, although the final list may be shorter.
     """
-    entry = True #Make sure we loop once
-    while entry :
+    entry = True  # Make sure we loop once
+    while entry:
         batch = []
-        while len(batch) < batch_size :
-            try :
+        while len(batch) < batch_size:
+            try:
                 entry = iterator.next()
-            except StopIteration :
+            except StopIteration:
                 entry = None
-            if entry is None :
-                #End of file
+            if entry is None:
+                # End of file
                 break
             batch.append(entry)
-        if batch :
+        if batch:
             yield batch
 ```
 
 Here is an example using this function to divide up a large FASTQ file,
-SRR014849.fastq (from this [compressed
-file](ftp://ftp.ncbi.nlm.nih.gov/sra/static/SRX003/SRX003639/SRR014849.fastq.gz)
-at the NCBI):
+`SRR014849_1.fastq` (from this [compressed
+file](https://github.com/biopython/biopython.github.io/blob/master/examples/SRR014849.fastq.gz)):
```

Comment:
Sadly the NCBI SRA has made getting hold of raw FASTQ files overly complicated, but the good news is the ENA does a much better job. How about we link to ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR014/SRR014849/SRR014849.fastq.gz (see also http://www.ebi.ac.uk/ena/data/view/SRR014849 as well) instead of putting a large FASTQ file into the website repository?